### DISCUSSION TITLE
Access parent on runway fieldtype

### DISCUSSION THREAD

**Discussion by martink635** [statamic-rad-pack/runway#378] [Help]
Currently it seems there is no way to access the parent from a Fieldtype in Runway?

```php
<?php

namespace App\Fieldtypes;

use Statamic\Entries\Entry;
use Statamic\Fields\Fieldtype;

class CustomFieldType extends Fieldtype
{
  ...

  public function preload()
  {
        dd($this->field->parent()); // this is always null
  }
    ...
}
```

Let me know if I am missing something. I want to access the ID of the entry/runway model we are editing, since we need to display some relevant information in a custom field type.

Thanks!

**Comment by duncanmcclean** [comment]
It’s not intentional - I’ve opened a bug report for it: #379

**Reply by duncanmcclean** [reply]
This should now be fixed in the latest version (v5.5.2).
